Extension/src/LanguageServer/client.ts

```diff
@@ -1799,9 +1799,36 @@ export class DefaultClient implements Client {
                             const candidate: string = response.candidates[i];
                             const tuUri: vscode.Uri = vscode.Uri.parse(candidate);
                             if (await provider.canProvideConfiguration(tuUri, tokenSource.token)) {
-                                const configs: SourceFileConfigurationItem[] = await provider.provideConfigurations([tuUri], tokenSource.token);
+                                const configs: util.Mutable<SourceFileConfigurationItem>[] = await provider.provideConfigurations([tuUri], tokenSource.token);
                                 if (configs && configs.length > 0 && configs[0]) {
-                                    return configs;
+                                    const fileConfiguration: configs.Configuration | undefined = this.configuration.CurrentConfiguration;
+                                    if (fileConfiguration?.mergeConfigurations ?? false) {
```

Comment:
The `?? false` seems unnecessary. Is there a reason to use that?

Reply:
just habit from C#, fixed :)